src/blog/2022/011-form-js-0.10.0.md

```diff
@@ -0,0 +1,141 @@
+
+<div class="figure full-size">
+  <a href="https://demo.bpmn.io/form">
+    <img src="{{ assets }}/attachments/blog/2022/011-text-view.gif">
+  </a>
+
+  <p class="caption">
+    You can reference the value of other form components to display in the text view component.
+  </p>
+</div>
+
+## Date and Time
+
+The date and time component allows your users to display and select a date, time, or both. It is fully accessible from the keyboard.
+
+<div class="figure full-size">
+  <a href="https://demo.bpmn.io/form">
+    <img src="{{ assets }}/attachments/blog/2022/011-date-time.gif">
+  </a>
+
+  <p class="caption">
+    The new date and time picker is designed for maximum convenience, whether accessed with the mouse or keyboard.
+  </p>
+</div>
+
+## Prefix and Suffix Adornments
+
+With the new prefix and suffix adornments you are able to enrich your components with additional information, and provide guidance for your users.
+
+<div class="figure full-size">
+  <a href="https://demo.bpmn.io/form">
+    <img src="{{ assets }}/attachments/blog/2022/011-adornments.gif">
+  </a>
+
+  <p class="caption">
+    Adornments provide guidance for users in a more convenient way than labels or hints.
+  </p>
+</div>
+
+## Decimal Numbers
+
+The number component now supports decimals. You are able to define the maximum allowed decimal digit and also the increment.
+
+<div class="figure full-size">
+  <a href="https://demo.bpmn.io/form">
+    <img src="{{ assets }}/attachments/blog/2022/011-decimals.gif">
+  </a>
+
+  <p class="caption">
+    Decimal number inputs are crucial for many form applications.
+  </p>
+</div>
+
+## Hiding Components Based on FEEL Expressions
+
+With this feature you are able to hide components based on variables and conditions.
+This allows you to build forms which are truly flexible and adaptive.
+
+<div class="figure full-size">
+  <a href="https://demo.bpmn.io/form">
+    <img src="{{ assets }}/attachments/blog/2022/011-conditional-rendering.gif">
+  </a>
+
+  <p class="caption">
+    FEEL & Conditional Rendering makes it easy to hide components based on easy-to-write, yet powerful rules.
+  </p>
+</div>
+
+## Validate and Preview Mode 
+
+With the new validate and preview mode you are provided with a live preview of your form. Furthermore you get access to the data input and output section which allows you to immediately validate your form with data.
+
+<div class="figure full-size">
+  <a href="https://demo.bpmn.io/form">
+    <img src="{{ assets }}/attachments/blog/2022/011-playground.gif">
+  </a>
+
+  <p class="caption">
+    The form preview is shown next to the form editor, allowing to immediately validate a form and its rules.
+  </p>
+</div>
+
+Try out the new features [on our demo](https://demo.bpmn.io/form) or using the [form-js playground](https://github.com/bpmn-io/form-js/tree/master/packages/form-js-playground).
```

Comment:
As @Skaiir mentioned (https://github.com/bpmn-io/bpmn.io/pull/155/files#r1056494740) this can be quite misleading as the playground version in the screenshot does not reflect the one mentioned in the link. 

If we want to promote the playground shown in the screenshot, we'd need to link [this one](https://github.com/camunda/form-playground) or improve our bpmn-io version (cf. https://github.com/bpmn-io/form-js/issues/328#issuecomment-1365841432).